src/features/unified-share-modal/ContactsField.js

```diff
@@ -28,6 +28,7 @@ type Props = {
     onInput?: Function,
     onPillCreate?: (pills: Array<SelectOptionProp | Contact>) => void,
     selectedContacts: Array<Contact>,
+    showInviteeAvatars?: boolean,
```

Comment:
actually, a better name here would be `showContactAvatars` probably, since other prop names deal with contacts and both components mention contact. I will see how that feels in practice